packages/recorder/index.js

```diff
@@ -27,27 +28,49 @@ export function addAsGlobal(ns = 'MediaRecorder') {
  * Note that the browser default is NOT used. This is because the support of the MediaRecorder API
  * is still not large.
  *
- * Another reason to not want to use the default is that the ITSLanguage backend currently only
- * supports WAVE as input.
+ * Another reason to not want to use the default is that the ITSLanguage backend
+ * currently only supports WAVE as input.
  *
  * @param {MediaStream} stream - Stream to record from.
- * @param {boolean} [setToWindow=false] - Override or set MediaRecorder to the window object,
- * defaults to false.
- * @param {string} [asObject='MediaRecorder'] - Optionally give the object another name then
- * MediaRecorder.
+ * @param {Object[]} [plugins = []] - Optionally an array with plugins to enable
+ * on the recorder created.
+ * @param {boolean} [setToWindow=false] - Override or set MediaRecorder to the
+ * window object.
+ * @param {string} [asObject='MediaRecorder'] - Optionally give the object
+ * another name than MediaRecorder.
  * @returns {MediaRecorder} - An instance of the created MediaRecorder.
  */
 export function createRecorder(
   stream = null,
+  plugins = [],
   setToWindow = false,
   asObject = 'MediaRecorder',
 ) {
   if (setToWindow) {
     addAsGlobal(asObject);
   }
 
+  // Create the MediaRecorder object.
   const recorder = new MediaRecorder(stream);
 
+  // Prepare the plugins object, here we store an instance of a plugin.
+  recorder.plugins = [];
+
+  // Enhance the recorder with some (or none) plugins.
+  if (plugins.length > 0) {
```

Comment:
This size check doesn’t really add anything. If the array is empty, it’s still fine to call `.forEach` on it.

Reply:
true!